src/App/Fossa/Config/Container/Analyze.hs

```diff
@@ -57,7 +59,10 @@ data ContainerAnalyzeConfig = ContainerAnalyzeConfig
   { scanDestination :: ScanDestination
   , revisionOverride :: OverrideProject
   , imageLocator :: ImageText
-  , usesExperimentalScanner :: Bool
+  , -- \* For Experimental Scanner
```

Comment:
Migrating some of the config/const values here, so it's easy to diagnose in FOSSA_CONFIG_DEBUG, debug bundle etc, and easier to provide override ux option in future